packages/ingest/src/FtpProviderClient.ts

```diff
@@ -200,20 +200,21 @@ class FtpProviderClient {
    * @returns {Promise.<{ s3uri: string, etag: string }>} an object containing
    *    the S3 URI and ETag of the destination file
    */
-  async sync(
-    remotePath: string,
-    bucket: string,
-    key: string
-  ): Promise<{s3uri: string, etag: string}> {
-    const remoteUrl = `ftp://${this.host}/${remotePath}`;
-    const s3uri = S3.buildS3Uri(bucket, key);
+  async sync(params: {
```

Comment:
Changes made to this class are entirely related to the parameterization of the .sync ProviderClient base class/this class  and/or refactors related to param names/etc.